.eslintrc.js

```diff
@@ -38,7 +38,7 @@ module.exports = {
           "before": false
       }],
       "comma-style": ["error", "last"],
-      "complexity": "error",
+      "complexity": ["error", { "max": 21 }],
```

Comment:
What was the point of editing this lint rule?

Reply:
@blackfalcon , could you please explain as this code was changed by you?

Reply:
Resetting the default to allow for a higher max ceiling from `20` to `21` so that we can release this update without needing a new refactoring of code.

Reply:
Won't this just get blown away the next time we auto-update this component from the generator?

Shouldn't we just disable or disable-line the rule where it is problematic in the file?